Library/Homebrew/cmd/test-bot.rb

```diff
@@ -520,7 +521,7 @@ def formula formula_name
               test "brew", "uninstall", "--force", *unchanged_build_dependencies
               unchanged_dependencies -= unchanged_build_dependencies
             end
-            test "brew", "install", bottle_filename
+            test "brew", "install", "--force-bottle", bottle_filename
```

Comment:
Just to be helpful, the test-bot seems to actually be rejecting bottles [now](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/30039/version=yosemite/console) because of the existing structure. Hmm.